web/global/input.scss

```diff
@@ -1,5 +1,12 @@
 .fci-input-container {
     padding-bottom: 16px;
+    label {
+        font-family: 'Roboto';
```

Comment:
I feel like we had that quite a lot in the recent PRs, should we unify? It's `scss`, so should be easy

Reply:
Yeah sure thing 👍